Add files via upload

dyganar opened the pull request:
Adicionado função que alerta usuário quando marcado no grupo.
Desabilitar /setprivacy para funcionar.

dyganar committed:
Add files via upload

TiagoDanin commented on bot.py:
```diff
@@ -1,73 +1,85 @@
+		inst_command_admin.unwarn(data=msg['data'])
+	else:
+		try:
+			#************************************************************
```
Não há necessidade de comentário (Sem nenhuma informação)

TiagoDanin commented on bot.py:
```diff
@@ -1,73 +1,85 @@
-
-	while 1:
-		sleep(100)
+import telepot
```
Use o EditorConfig (Esse projeto é configurado para usar Tabs e LF)
Veja em https://github.com/Programeiros/TycotBot/#como-contribuir

TiagoDanin commented on bot_command.py:
```diff
@@ -1,247 +1,278 @@
+
+
+class command_user(control, keyboard):	
+	#***************************************************************************************************
```
O mesmo aqui (comentário sem nenhuma informação)

dyganar replied:
Eu uso comentários para delimitar onde ocorreu as alterações facilitando assim a análise. Não tinha visto sobre o EditorConf ^^'

TiagoDanin reviewed (changes requested):
- Caso o usuário troque seu username ele não sera notificado.
- A estrutura do código não respeita os padrões do EditorConfig.
- Uso de comentário no código desnecessário. 

Por favor verifique suas alterações.

E seja bem vindo ao Github 😄 🎉

TiagoDanin commented:
Regex funciona muito bem: https://regex101.com/r/iy7NMx/1

senegalha commented:
No aguardo das correções propostas pelo @TiagoDanin

dyganar commented:
Sobre o usuário modificar o username ele terá que reativar o alerta já que o banco está armazenando os dados no momento da entrada no grupo ou ativação do alerta. Vou pensar em um método para verificar se houve mudança no username.

senegalha commented:
@Takanar Use o ID do usuário, é melhor. 

ID é um só

TiagoDanin commented:
@dansenpir mas é preciso do username atual para verifica.
Recomendo salva, toda fez que um usuário usar um comando do bot.

senegalha commented:
@TiagoDanin Não sabia disso.

É uma boa essa ideia.

Mas seria necessário algum comando que fosse fácil, rápido, mas de uso frequente, para sempre ir renovando o username.

Seria possível adicionar isso no Inline Keyboard onde estará o link para ter acesso a mensagem?
Assim, sempre que clicar, ele iria a mensagem e também atualizaria o username.

dyganar committed:
Please enter the commit message for your changes. Lines starting
 with '#' will be ignored, and an empty message aborts the commit.
 On branch issue47
 Changes to be committed:
	modified:   bot.py
	modified:   bot_command.py
	modified:   sql.py

dyganar committed:
On branch issue47
 Changes to be committed:
	modified:   bot_command.py
	modified:   sql.py

dyganar committed:
Merge branch 'tk' of https://github.com/Takanar/TycotBot into tk

 Changes to be committed:
	modified:   bot.py
	modified:   bot_command.py
	modified:   sql.py

dyganar committed:
Update bot.py

dyganar committed:
Update bot_command.py

dyganar committed:
Update sql.py
